src/hb-style.cc

```diff
@@ -89,13 +61,13 @@ _hb_ratio_to_angle (float r)
  * @font: a #hb_font_t object.
  * @style_tag: a style tag.
  *
- * Searches variation axes of a hb_font_t object for a specific axis first,
+ * Searches variation axes of a #hb_font_t object for a specific axis first,
  * if not set, then tries to get default style values from different
  * tables of the font.
  *
  * Returns: Corresponding axis or default value to a style tag.
  *
- * Since: EXPERIMENTAL
+ * Since: REPLACEME
  **/
 float
 hb_style_get_value (hb_font_t *font, hb_tag_t tag)
```

Comment:
Shouldn’t this be `hb_style_tag_t` instead of plain `hb_tag_t`?

Reply:
Thanks for this! It supposed to be `hb_style_tag_t` for sure, I couldn't use it as the enum wasn't in public header to not confuse the doc gen.

Reply:
Done. Thanks all.